### DISCUSSION TITLE
Landmark Detection

### DISCUSSION THREAD

**Discussion by CometManAtGitHub** [MIC-DKFZ/nnUNet#804] [Ideas]
Hi,

are you planning to add landmark detection? Maybe the segmentation backbones features maps can be used to extract landmarks. Also the segmentation may benefit from this additional learning task (multi-task-learning).

 Best regards.

**Comment by FabianIsensee** [comment]
Hi, this is out of scope for nnU-Net. You can formulate landmark detection as a segmentation problem and then use nnU-Net but standard landmark detection is not planned.
Best,
Fabian
